ElectronNET.API/App.cs

```diff
@@ -464,40 +464,47 @@ public void Show()
         /// The current application directory.
         /// </summary>
         /// <returns></returns>
-        public async Task<string> GetAppPathAsync()
+        public async Task<string> GetAppPathAsync(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
With CSharp 7.1 support this could be reduced to `= default` but I didn't want to change too many things at the same time